packages/page-settings/src/index.tsx

```diff
@@ -30,7 +29,6 @@ function SettingsApp ({ basePath, onStatusChange }: Props): React.ReactElement<P
       text: t('General')
     },
     {
-      count: numExtensions,
```

Comment:
Why do we remove this?

Reply:
It was removed when the popup was added, has been reinstated now